src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -758,6 +760,21 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
         $cache->write($rootCode, $container->getResources());
     }
 
+    /**
+     * Returns a proxy dumper implementing the
+     * {@see \Symfony\Component\DependencyInjection\LazyProxy\PhpDumper\DumperInterface} DumperInterface or null.
+     *
+     * @return DumperInterface|null
```

Comment:
the full dobcblock should be removed as it doesn't provide anything more than the signature.